src/plugins/abrt-action-generate-machine-id

```diff
@@ -52,7 +52,10 @@ def generate_machine_id_dmidecode():
 
     # Run dmidecode command
     for k in keys:
-        data = check_output(["dmidecode", "-s", k]).strip()
+        try:
+            data = check_output(["dmidecode", "-s", k]).strip()
+        except (OSError, CalledProcessError) as ex:
+            raise RuntimeError("Execution of dmidecode failed: {0}".format(str(ex)))
```

Comment:
Please, suggest a better error message.

Reply:
It doesn’t matter too much. What about including `stdout` or `stderr`?

https://github.com/python/cpython/blob/b0a2c0fa83f9b79616ccf451687096542de1e6f8/Lib/subprocess.py#L112